src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -176,6 +192,20 @@ private static void getAllFileInfo(RepoConfiguration config, Path directory, Lis
                     continue;
                 }
 
+                if (!FileUtil.isValidPath(filePath.toString())) {
+                    logger.warning(String.format(INVALID_FILE_PATH_MESSAGE_FORMAT, filePath));
+                    continue;
+                }
+
+                if (!fileExistsInRepo(relativePath, Paths.get(config.getRepoRoot()))) {
+                    continue;
+                }
+
+                if (isBinaryFile(relativePath, Paths.get(config.getRepoRoot()))) {
+                    logger.log(Level.FINE, relativePath + " is a binary file and will be ignored.");
```

Comment:
This warning will spam the console output. For now, maybe a better solution is to check the file is inside format list first, then, we check if it is binary.

Reply:
Will remove the logging part. 




> For now, maybe a better solution is to check the file is inside format list first, then, we check if it is binary.

Because we are going to implement "detect all file formats" in #708 , we might need to check for all files. Also, even if file formats are specified by users, some files might still be binary,  regardless of the file format (for rare cases). So I think it might be better if all files that come in gets checked for binary files, regardless of the white-listed file format list.